templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -44,16 +42,36 @@ async def chat(
         )
         for m in data.messages
     ]
+    return last_message, messages
```

Comment:
Here we're a bit in a pickle as last_message is a different message type than messages, to be consistent we could either return:

```suggestion
    return last_message.content, messages
```
or
```suggestion
    return ChatMessage(role=last_message.role,
            content=last_message.content), messages
```

I think the first one is easier to understand